spec/models/partner_spec.rb

```diff
@@ -114,6 +114,20 @@
     end
   end
 
+  describe "#partials_to_show" do
+    let(:partner) { create(:partner, diaper_bank_id: 100) }
+    it 'has 9 partials when there are no displayable partials configured' do
+      expect(partner.partials_to_show.size).to eq(9)
+    end
+
+    it 'displays the number of displayable partials when they are configured' do
+      partner.diaper_bank_id = 100
+      FactoryBot.create(:partner_form, diaper_bank_id: 100,
```

Comment:
```suggestion
      FactoryBot.create(:partner_form, diaper_bank_id: partner.id,
```
Since partner is already defined above with `let` could we just use the partner from there?